.evergreen/atlas_data_lake/README.md

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Suggested title: `# Atlas Data Lake Testing`

Reply:
I have added the title.